pkg/overlay/config.go

```diff
@@ -41,7 +41,7 @@ type NodeSelectionConfig struct {
 	NewNodePercentage float64       `help:"the percentage of new nodes allowed per request" default:"0.05"` // TODO: fix, this is not percentage, it's ratio
 	MinimumVersion    string        `help:"the minimum node software version for node selection queries" default:""`
 	OnlineWindow      time.Duration `help:"the amount of time without seeing a node before its considered offline" default:"1h"`
-	DistinctIP        bool          `help:"require distinct IPs when choosing nodes for upload" releaseDefault:"true" devDefault:"false"`
+	DistinctIP        bool          `help:"require distinct IPs when choosing nodes for upload" releaseDefault:"false" devDefault:"false"`
```

Comment:
Please keep true here. I have tested the IP filter. It doesn't work the way we need it but it is still an inprovement. I missed to activate it in production.

Reply:
Agreeing here as well!

Reply:
okay, so, this should stay true, and we should activate in prod. that's kind of what i thought